broker/l2tp_broker.py

```diff
@@ -772,7 +773,7 @@ def hook(self, name, *args):
       return
 
     # Execute the registered hook
-    logger.debug("Executing hook '%s' via script '%s'..." % (name, script))
+    logger.debug("Executing hook '%s' via script '%s'... %s" % (name, script, str([str(x) for x in args])))
```

Comment:
Can you move the last `%s` to be placed next to the script name? So something like `via script '%s %s'...`.